gstats/gstats.c

```diff
@@ -1009,7 +1009,7 @@ static int RecvSessionKey()
 			return GE_DATAERROR; //temp fix in case len == -1, SOCKET_ERROR
 	}
 
-	resp[len - 1] = '\0';
+	resp[len] = '\0';
```

Comment:
Looks like i made mistake and _**truncate**_ last byte every time.

Now should work as expected with 128 bytes max.